CONTRIBUTING.md

```diff
@@ -2,36 +2,18 @@
 
 nix.dev is a community effort to collect, create, and maintain world-class learning resources for Nix.
 
-We strongly encourage everyone interested to contribute by asking informed questions or directly proposing changes.
+We strongly encourage everyone interested to participate:
+- Make a [pull request](https://github.com/NixOS/nix.dev/pulls) if you want to introduce an incremental change.
+- Open an [issue](https://github.com/NixOS/nix.dev/issues) if you want to discuss a significant change before starting to work on it.
 
-Make a [pull request](https://github.com/NixOS/nix.dev/pulls) if you want to introduce an incremental change.
-Note our [considerations on licensing and attribution](#licensing-and-attribution).
-
-Open an [issue](https://github.com/NixOS/nix.dev/issues) if you want to clarify something not evident from what is provided in this repository, or if you want to discuss a significant change before starting to work on it.
-
-## Code of conduct
-
-Adapted from the [Contributor Covenant] and [The Carpentries Code of Conduct]:
-
-- Use welcoming and inclusive language
-- Show empathy and respect towards other people
-- Be respectful of different viewpoints and experiences
-- Give and gracefully accept constructive criticism
-- Focus on what is best for the community
```

Comment:
There's no code of conduct on the contributing page (which is notably also linked to from nixos.org). We do have https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md though, which is accessible from all the repos, so I think it's fine

Reply:
I'll add that in a follow-up PR, since the contributor guides need a bit of clean-up as well.